Tests/Serializer/CallbackTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use \FOS\ElasticaBundle\Serializer\Callback;
+use \Symfony\Component\Serializer\Serializer;
+
+class CallbackTest extends PHPUnit_Framework_TestCase
+{
+    public function testSerializerMustHaveSerializeMethod()
+    {
+        $callback = new Callback();
+        $this->setExpectedException(RuntimeException::class, 'The serializer must have a "serialize" method.');
+        $callback->setSerializer(new \stdClass());
+    }
+
+    public function testSetGroupsWorksWithValidSerializer()
+    {
+        $callback = new Callback();
+        $serializer = $this->prophesize(Serializer::class);
```

Comment:
@giovannialbero1992 Ah, I should have been paying more attention to the Travis builds. The test was working fine for me locally. I changed it to use `getMock` instead, fingers crossed the build succeeds